k8sfv/leastsquares_test.go

```diff
@@ -22,6 +22,9 @@ import (
 )
 
 var _ = Context("least squares", func() {
+	BeforeEach(func() {
+		expectFelixReady()
```

Comment:
These particular tests don't involve Felix, I believe.  So I don't think they should check that Felix is ready.